tox.ini

```diff
@@ -2,13 +2,15 @@
 # fakeroot -u tox --recreate
 
 [tox]
-envlist = py32, py33, py34, py35
+envlist = py{32,33,34,35}{,-attic}
```

Comment:
if that means to build double amount of test envs, we should not do that, it already takes rather long right now, without that.